tools/hdf_converter/hdf_converter.xml

```diff
@@ -1,7 +1,8 @@
 <tool id="apCLMS_to_RamClustR_converter" name="apLCMS to RamClustR converter" version="0.0.1" python_template_version="3.5">
     <description>converts aplcms HDF output to RamClustR csv input</description>
```

Comment:
Is this true every time or is this applicable only to our wrappers of said tools? Maybe we could link them in `help` section to clarify this.